src/main.rs

```diff
@@ -266,6 +267,7 @@ async fn start(cfg: &Config) -> ProtocolResult<()> {
         consensus_interval: cfg.consensus.interval,
     };
 
+    init_tracer(my_address.as_hex()).unwrap();
```

Comment:
Use `init().expect(error message)` here.

Reply:
> Use `init().expect(error message)` here.

returned ProtocolResult now, the `unwrap` is similar to `expect`